client/app/resources/collections-api.factory.js

```diff
@@ -58,6 +58,9 @@
       options = options || {};
 
       if (options.expand) {
+        if (angular.isArray(options.expand)) {
+          options.expand = options.expand.join(',');
+        }
```

Comment:
So you're converting an Array to a String here. Is that necessary for the API to work correctly if we pass multiple `expand` parameters?
`expand: ['resources', 'service_requests']`